content/03-pytest.md

```diff
@@ -117,27 +117,57 @@ whether this works on Windows):
    $ git add example.py
    $ git commit
    ```
-5. Git add and commit the file:
-   ```shell
-   $ git add example.py
-   $ git commit
-   ```
-6. Create the following shell script. Save this file as `pre-commit`:
+5. Create the following shell script. Save this file as `pre-commit`:
    ```shell
    #!/bin/bash
 
    pytest example.py
    ```
-7. Make `pre-commit` executable:
-   ```shell
-   $ chmod +x pre-commit
-   ```
+6. Make `pre-commit` executable:
+      `````{tabs}
+         ````{tab} Bash
+     
+            ```bash
+            $ chmod +x pre-commit
+            ```
+         ````
+         ````{tab} Anaconda prompt
+     
+            Nothing to do.
+         ````
+      `````
+7. Check the hook script:
+      `````{tabs}
+         ````{tab} Bash
+     
+            ```bash
+            $ ./pre-commit
+            ```
+         ````
+         ````{tab} Anaconda prompt
+     
+            ```bash
+            $ ./pre-commit.bat
```

Comment:
I don't think it says name this `pre-commit.bat` on Windows - or is this implicit somewhere?

Reply:
I picked that up from https://github.com/coderefinery/testing/issues/58. I don't really know :-)

Reply:
I misunderstood the question. I will add something ...